src/components/v5/common/Fields/TextareaBase/TextareaBase.tsx

```diff
@@ -17,8 +19,8 @@ const TextareaBase = React.forwardRef<HTMLTextAreaElement, TextareaBaseProps>(
       value,
       stateClassNames: stateClassNamesProp,
       wrapperClassName,
-      maxCharNumber,
       message,
+      maxLength,
```

Comment:
do we want to add maxLength to the textarea too?

Reply:
@bassgeta  yes, the same as above

Reply:
sorry, misunderstaning, I mean that if we could add the maxLength to the `<input />` element as an HTML property too